nixos/modules/services/misc/mediatomb.nix

```diff
@@ -206,64 +259,107 @@ in {
 
       dataDir = mkOption {
         type = types.path;
-        default = "/var/lib/mediatomb";
+        default = "/var/lib/${name}";
+        description = ''
+          The directory where ${cfg.serverName} stores its state, data, etc.
```

Comment:
Unfortunately I just noticed this post-merge. How can you refer to the value of an option within an option description? Have you tried building the docs with this? I suspect this will just always expand to the default value.